server/generate/srcfiles.go

```diff
@@ -90,10 +90,62 @@ var (
 		"shell/pty/pty_unsupported.go",
 
 		"pivots/pivots.go",
-		"pivots/namedpipe/namedpipe.go",
-		"pivots/namedpipe/npipe_windows.go",
-		"pivots/namedpipe/znpipe_windows_386.go",
-		"pivots/namedpipe/znpipe_windows_amd64.go",
+		"pivots/winio/reparse.go",
```

Comment:
We can remove a lot of stuff from this list. I ran some tests, and this seems to work:
```
		"pivots/pivots.go",
		"pivots/winio/reparse.go",
		// "pivots/winio/tools/etw-provider-gen/main.go",
		"pivots/winio/backup.go",
		// "pivots/winio/archive/tar/stat_atim.go",
		// "pivots/winio/archive/tar/stat_atimespec.go",
		// "pivots/winio/archive/tar/writer.go",
		// "pivots/winio/archive/tar/common.go",
		// "pivots/winio/archive/tar/stat_unix.go",
		// "pivots/winio/archive/tar/reader.go",
		"pivots/winio/hvsock.go",
		"pivots/winio/privilege.go",
		"pivots/winio/syscall.go",
		// "pivots/winio/backuptar/tar.go",
		// "pivots/winio/backuptar/noop.go",
		"pivots/winio/zsyscall_windows.go",
		"pivots/winio/file.go",
		// "pivots/winio/wim/decompress.go",
		// "pivots/winio/wim/validate/validate.go",
		// "pivots/winio/wim/lzx/lzx.go",
		// "pivots/winio/wim/wim.go",
		"pivots/winio/fileinfo.go",
		"pivots/winio/ea.go",
		"pivots/winio/pipe.go",
		"pivots/winio/sd.go",
		"pivots/winio/vhd/vhd.go",
		//"pivots/winio/vhd/mksyscall_windows.go",
		"pivots/winio/vhd/zvhd.go",
		"pivots/winio/pkg/security/syscall_windows.go",
		//"pivots/winio/pkg/security/mksyscall_windows.go",
		"pivots/winio/pkg/security/zsyscall_windows.go",
		"pivots/winio/pkg/security/grantvmgroupaccess.go",
		// "pivots/winio/pkg/etw/provider.go",
		// "pivots/winio/pkg/etw/eventmetadata.go",
		// "pivots/winio/pkg/etw/newprovider_unsupported.go",
		// "pivots/winio/pkg/etw/eventdata.go",
		// "pivots/winio/pkg/etw/eventopt.go",
		// "pivots/winio/pkg/etw/wrapper_64.go",
		// "pivots/winio/pkg/etw/eventdescriptor.go",
		// "pivots/winio/pkg/etw/ptr64_32.go",
		// "pivots/winio/pkg/etw/providerglobal.go",
		// "pivots/winio/pkg/etw/fieldopt.go",
		//"pivots/winio/pkg/etw/mksyscall_windows.go",
		// "pivots/winio/pkg/etw/zsyscall_windows.go",
		// "pivots/winio/pkg/etw/ptr64_64.go",
		// "pivots/winio/pkg/etw/sample/sample.go",
		// "pivots/winio/pkg/etw/eventdatadescriptor.go",
		// "pivots/winio/pkg/etw/etw.go",
		// "pivots/winio/pkg/etw/newprovider.go",
		// "pivots/winio/pkg/etw/wrapper_32.go",
		"pivots/winio/pkg/guid/guid.go",
		"pivots/winio/pkg/fs/fs_windows.go",
		"pivots/winio/pkg/process/process.go",
		"pivots/winio/pkg/process/syscall.go",
		"pivots/winio/pkg/process/zsyscall_windows.go",
		// "pivots/winio/pkg/etwlogrus/hook.go",
		"pivots/winio/namedpipe.go",
```

Reply:
done